initialize-project

```diff
@@ -55,12 +55,14 @@ git_clone() {
 
   local forking_org
   forking_org="$(forking_org)"
-  if [[ -n "${forking_org}" ]] && [[ "${forking_org}" != "https://github.com/jspecify" ]]; then
+  if [[ -n "${forking_org}" ]] \
+    && [[ "${forking_org}" != "https://github.com/jspecify" ]] \
+    && [[ "${forking_org}" != "git@github.com:/jspecify" ]] ; then
```

Comment:
```suggestion
    && [[ "${forking_org}" != "git@github.com:jspecify" ]] ; then
```

Reply:
Ugh, that's because I've used that URL for my clone... I agree the shorter one is what should be used.